projects/sm-pwr/bop/steamer.py

```diff
@@ -535,8 +570,14 @@ def __heat_sink_rate(self, water_p, water_s):
 
         # Total area of heat tranfer
         area = 2*math.pi*radius_mean* self.n_helicoil_tubes * self.helicoil_length
-
-        q_p = - area * 1/one_over_U * (temp_p - temp_s)
```

Comment:
As you pointed out this is incorrect. Maybe average the temperatures on the primary and secondary:
(temp_p+temp_p_in)/2 - (temp_s+temp_s_in)/2